tools/ibc-test/src/tests/channel.rs

```diff
@@ -0,0 +1,89 @@
+use std::str::FromStr;
+
+use crate::framework::{binary::channel::run_arbitrary_binary_channel_test, utils::ckb::*};
+use ckb_types::H256;
+use ibc_test_framework::{chain::chain_type::ChainType, prelude::*};
+
+#[test]
+fn test_channel() -> Result<(), Error> {
+    run_arbitrary_binary_channel_test(&ChannelTest)
```

Comment:
The test code is more complicated and covers a lot of logic. See if you can add some comments and supplement the assertion of the test code